app/pug/user_profile.pug

```diff
@@ -44,10 +44,10 @@ block content
         
         dt Roles
         dd
-          .monospace #{JSON.stringify(user.roles)}
+          .monospace #{JSON.stringify(user.roles).replace(/"/g, "").replace(/,/g, ", ").replace(/^\[|]/g, '')}
```

Comment:
Seeing as these regexes are used across files, might be more maintainable to create a helper for this purpose

Reply:
Added a new pug mixin to cover all of the above `.replace` simplifications.